src/core/src/sys/interactions.rs

```diff
@@ -443,4 +445,17 @@ mod test {
         interactions.add_global(Box::new(Wolf::new(1.0)));
         assert_eq!(interactions.globals().len(), 1);
     }
+
+    #[test]
+    fn test_globals_cutoff() {
```

Comment:
Why is this test needed? We should have tests on `Resize` instead, checking that we get the right cutoff in specific cases:

- no cutoff
- only pairs
- only globals
- both

Or we could make this a function of `Interactions` and test it here.